services-js/registry-certs/client/birth/questions/QuestionComponent.tsx

```diff
@@ -24,12 +24,12 @@ export default function QuestionComponent(props: Props): JSX.Element {
       <div className={QUESTION_CONTAINER_STYLING}>{props.children}</div>
 
       <div className="g g--r m-t700">
-        <div className="g--6 ta-r m-b500">
+        <div className="g--6 ta-r m-v500">
           {props.handleProceed &&
             !props.startOver && (
               <button
                 type="button"
-                className="btn btn--br btn--b-sm"
+                className="btn btn--b-sm"
```

Comment:
Per Sebastian, we should switch away from the bordered Next buttons all across the app.

Can you switch the checkout flow back buttons to links as well?

Reply:
Thanks for the catch!